docs/source/usage.rst

```diff
@@ -0,0 +1,185 @@
+Usage
+=====
+
+.. contents::
+
+Most common options
+-------------------
+
+The following exemple uses all most common options. 
+It will add a link to your project website in all pages header and show a link to source code aside every documented elements. 
+
+::
+
+    pydoctor \
+        --add-package=(packagedirectory) \
+        --project-name=(name) \
+        --project-url=https://github.com/(user)/(repo)/ \
+        --html-viewsource-base https://github.com/(user)/(repo)/tree/(tag_or_commit_sha) \
+        --make-html \
+        --html-output=apidocs \
+        --project-base-dir "$(pwd)" \
+        --docformat=restructuredtext 
+
+.. note:: This exemple assume you are running the ``pydoctor`` build from Unix and the current directory is the root folder of your package. 
+
+.. warning:: The ``--html-viewsource-base`` argument  should point to a tag or a commit SHA rather than a branch since line 
+    numbers aren't going to match otherwise when commits are added to the branch after the documentation has been published.
+
+Publish your documentation with Github actions
```

Comment:
not sure if we should keep this.

This is just standard static HTML file publishing .

So maybe just mention that pydoctor output are static HTML pages without no extra server-side support.
People should know what to do with them

Reply:
If you are talking about the whole Github action exemple, I think we should keep it as it makes it so easy to publish pydoctor docs by copy/pasting this action and gives exemple to users. 

The trick with `--project-url=https://github.com/$GITHUB_REPOSITORY` and `--html-viewsource-base=https://github.com/$GITHUB_REPOSITORY/tree/$GITHUB_SHA` are not obvious to everybody 

Let's keep it please